s3-proxy/nginx.conf

```diff
@@ -31,7 +31,7 @@ http {
             proxy_request_buffering off;
             proxy_ssl_verify on;
             proxy_ssl_verify_depth 5;
-            proxy_ssl_trusted_certificate /etc/nginx/certs.pem;
+            proxy_ssl_trusted_certificate /etc/ssl/certs/ca-bundle.trust.crt;
```

Comment:
This one is a change that I don't fully understand, and not even sure how to test properly.

We used to `cat /etc/ssl/certs/*.pem > /etc/nginx/certs.pem` - but the latest version of `amazonlinux` doesn't even have those files anymore. Instead, it has:

```
/etc/ssl/certs/ca-bundle.crt
/etc/ssl/certs/ca-bundle.trust.crt
```

I don't understand the difference; the only thing I can find is this: https://serverfault.com/questions/620003/difference-between-ca-bundle-crt-and-ca-bundle-trust-crt It implies that we should use `ca-bundle.crt`, unless we really care about security? But maybe we do?

Reply:
Oh, forgot that we potentially append custom certs to /etc/nginx/certs.pem. Need to undo this.

Reply:
Actually, amazonlinux:2023 only has one bundle, so that makes it easy...